BLUESPAWN-client/src/util/configurations/CollectInfo.cpp

```diff
@@ -0,0 +1,62 @@
+#include <windows.h>
+
+#define SECURITY_WIN32
+#include <Security.h>
+
+#include "util/configurations/CollectInfo.h"
+#include "util/configurations/Registry.h"
+#include "util/log/Log.h"
+
+void OutputComputerInformation() {
+	LOG_INFO("Computer Information\n");
+	LOG_INFO("DNS FQDN: " << GetFQDN());
+	LOG_INFO("Computer DNS Name: " << GetComputerDNSName());
+	LOG_INFO("Active Directory Domain: " << GetDomain());
+	LOG_INFO("Operating System: " << GetOSVersion());
+	LOG_INFO("Current User: " << GetCurrentUser());
+}
+
+std::wstring GetOSVersion() {
+	return *Registry::RegistryKey(HKEY_LOCAL_MACHINE, L"SOFTWARE\\Microsoft\\WIndows NT\\CurrentVersion").GetValue<std::wstring>(L"ProductName");
+}
+
+std::wstring GetComputerDNSName() {
+	LPWSTR buffer = new WCHAR[256];
+	DWORD dwSize = sizeof(buffer);
+
+	//enum info: https://docs.microsoft.com/en-us/windows/desktop/api/sysinfoapi/ne-sysinfoapi-_computer_name_format
+	bool status = GetComputerNameEx(ComputerNamePhysicalDnsHostname, buffer, &dwSize);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'status' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=4994516)